kv: repro branch for closedts assertion

tbg opened the pull request:
- roachtest: tpccbench repro branch
- roachprod: use exit-on-error:false for crdb log cfg

Commit:
vendor: Bump pebble to b49f2eeba24b

Changes pulled in:

```
b49f2eeba24bb225582de9dbd71e35d645960256 *: Skip first iteration of read sample
bc446105db6a3ab399bc9c273a7101ca7bc758bb db: optimize levelIter for non-matching bloom filter
68086bf1b3a17bdecde7678f795be746d8feedc6 db: enhance Iterator with limited iteration mode
```

Release note (performance improvement): Address a performance regression
from a past change regarding read-triggered compactions.

Commit:
vendor: Bump pebble to e3809b89b488b6ebba340449a2a024324310df2a

Changes pulled in:
```
e3809b89b488b6ebba340449a2a024324310df2a *: Sample some first iter reads based on size read
b618819945bd0ddb5bb91723b009ee41079ff344 tool: fix find to support custom comparers and mergers
e20c3b203d2894ba94dbd3dad2e7b4231d44b865 tool: add FS option
```

Release note: None.

tbg committed:
roachprod: use exit-on-error:false for crdb log cfg

See https://github.com/cockroachdb/cockroach/pull/62763.

We seem to frequently miss the runtime errors resulting from
out-of-memory conditions in the stderr logs. We don't understand
exactly why yet, but it is very likely that with `exit-on-error`
(which is true by default) we are hitting errors outputting to
the sink which then kill the process before the runtime errors
bubble up.

While we develop a proper fix, avoid the problematic configuration
on roachprod clusters, which notably includes roachtests.

Release note: None

tbg committed:
roachtest: tpccbench repro branch

For https://github.com/cockroachdb/cockroach/issues/62655#issuecomment-809048210

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/63533)
<!-- Reviewable:end -->

tbg committed:
crash better

We want to close over the scope of the variables describing the crash,
this wasn't true previously. Basically nothing was reachable from the
crashing goroutine's scope.

Release note: None

tbg committed:
Go back to LoadWarehouses

Line searcher panics when the max is a passing run.

Release note: None

tbg committed:
print a read summary

r307 from the n5 repro
